code/src/cljs/sixsq/nuvla/ui/i18n/dictionary.cljs

```diff
@@ -419,6 +419,7 @@
         :login-link                                    "Login."
         :login-to                                      "Login to "
         :logo-url-placeholder                          "e.g. http://example.com/images/logo.png"
+        :logo-hint                                     "Images look best if in PNG or SVG format with a size of 282px x 200px."
```

Comment:
What about this more complete sentence:
"For an optimal display, we recommend using images in either PNG or SVG format with dimensions of 282 pixels width by 200 pixels height."

"Pour un affichage optimal, nous vous recommandons d'utiliser des images au format PNG ou SVG avec des dimensions de 282 pixels de largeur sur 200 pixels de hauteur."